swatch-model-events/schemas/event.yaml

```diff
@@ -94,6 +124,11 @@ properties:
           description: Metric ID for the measurement. (e.g. Cores, Sockets)
```

Comment:
Should there be information about what this is being matched up to in swatch product-config?

Reply:
Yeah I'll elaborate on that more, thanks!